src/drivers/net/ena.c

```diff
@@ -350,6 +350,89 @@ static int ena_admin ( struct ena_nic *ena, union ena_aq_req *req,
 	return rc;
 }
 
+/**
+ * Set async event notification queue config
+ *
+ * @v ena		ENA device
+ * @ret rc		Return status code
+ */
+static int ena_set_aenq_config ( struct ena_nic *ena,
+				 uint32_t enabled_groups ) {
+	union ena_aq_req *req;
+	union ena_acq_rsp *rsp;
+	union ena_feature *feature;
+	int rc = 0;
+
+	/* Construct request */
+	req = ena_admin_req ( ena );
+	req->header.opcode = ENA_SET_FEATURE;
+	req->set_feature.id = ENA_AENQ_CONFIG;
+	feature = &req->set_feature.feature;
+	feature->aenq.enabled_groups = cpu_to_le32 ( enabled_groups );
+
+	/* Issue request */
+	if ( ( rc = ena_admin ( ena, req, &rsp ) ) != 0 )
+		DBGC ( ena, "ENA %p failed to set AENQ config\n", ena );
+
+	return rc;
+}
+
+
+/**
+ * Create async queues
```

Comment:
"Create async event queue"?  Seems to be handling only a single queue, as far as I can tell.